csharp_dotnetcore/12.NLP-With-LUIS/LuisBot/LuisBot.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
+
+namespace Luis_Bot
+{
+    /// <summary>
+    /// Represents a bot that can process incoming activities.
+    /// For each interaction from the user, an instance of this class is called.
+    /// This is a Transient lifetime service.  Transient lifetime services are created
+    /// each time they're requested. For each <see cref="Activity"/> message received,
+    /// a new instance of this class is created. Objects that are expensive to construct,
+    /// or have a lifetime beyond the single Turn, should be carefully managed.
```

Comment:
This comment.... it's perfect. I'm weeping! 👍

Reply:
Thanks!